.travis.yml

```diff
@@ -36,8 +36,10 @@ jobs:
       php: 7.3
       env: COMPOSER_FLAGS=""
       script:
+        - rm -rf vendor/
         - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
         - export REPO=$DOCKER_USERNAME/phpinsights
         - docker build -f docker/Dockerfile -t $REPO:$TRAVIS_TAG .
         - docker tag $REPO:$TRAVIS_TAG $REPO:latest
-        - docker push $REPO
+        - docker push $REPO:$TRAVIS_TAG
+        - docker push $REPO:latest
```

Comment:
As far I know, `docker push $REPO` push all local tags at once, no ?

Reply:
To my knowledge, we can do a test =)
If we look at the phpinsights dockerhub a lot of tags are missing

Reply:
Yes, the current available build on docker hub was a manually build.

I tried to add the automatic build in #160 but since, we always had problems with environment variables.

Anyway, your improvement is okay for me, and we have to found a way to fix environments variables.

Maybe it'll be ok for the next release :crossed_fingers:

Reply:
Sorry, but I still had a little doubt about that
> we have to found a way to fix environments variables.

After this fix, what is the problem with environment variables ?